### PR TITLE
revert jakarta

### PR DESCRIPTION
**Pull request by ahmadshadab**

### CODE REVIEW

File: common-rest/src/main/java/org/uniprot/api/rest/output/converter/AbstractUUWHttpMessageConverter.java [hunk 1]
```diff
@@ -237,7 +237,7 @@ private void logStats(int counter, Instant start) {
         long millisDuration = Duration.between(start, now).toMillis();
         int secDuration = (int) millisDuration / 1000;
         String rate = String.format("%.2f", ((double) counter) / secDuration);
-        log.debug(
+        log.info(
```
Comment (LeonardoGonzales) [inline common-rest/src/main/java/org/uniprot/api/rest/output/converter/AbstractUUWHttpMessageConverter.java:240]: We can keep it debug, what do you think?

File: common-rest/src/main/java/org/uniprot/api/rest/output/converter/ErrorMessageXMLConverter.java [hunk 2]
```diff
@@ -4,27 +4,26 @@
...
-import jakarta.xml.bind.Marshaller;
-
 /** @author lgonzales */
 public class ErrorMessageXMLConverter extends AbstractGenericHttpMessageConverter<ErrorInfo> {
 
     private final Marshaller marshaller;
 
     public ErrorMessageXMLConverter() {
-        super(MediaType.APPLICATION_XML, UniProtMediaType.RDF_MEDIA_TYPE);
+        super(MediaType.APPLICATION_XML);
```
Comment (LeonardoGonzales) [inline common-rest/src/main/java/org/uniprot/api/rest/output/converter/ErrorMessageXMLConverter.java:26]: For RDF, why do you think we should remove it?

**Review by LeonardoGonzales — APPROVED** [review]
I have 2 small details, the RDF one I am not sure.

File: common-rest/src/main/java/org/uniprot/api/rest/output/converter/AbstractUUWHttpMessageConverter.java (hunk 1 above)
Comment (ahmadshadab) [reply common-rest/src/main/java/org/uniprot/api/rest/output/converter/AbstractUUWHttpMessageConverter.java]: Yes, changed it to debug

File: common-rest/src/main/java/org/uniprot/api/rest/output/converter/ErrorMessageXMLConverter.java (hunk 2 above)
Comment (ahmadshadab) [reply common-rest/src/main/java/org/uniprot/api/rest/output/converter/ErrorMessageXMLConverter.java]: Added it back

**Pull request merged**